tests/test_checker.py

```diff
@@ -9,6 +9,12 @@
 
 import ntia_conformance_checker.sbom_checker as sbom_checker
 
+import logging
+
+logging.basicConfig(
+    format='%(asctime)s - %(levelname)s - %(message)s', level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
Why add logging? I'm not strongly opposed, but I don't personally understand the benefit to project developers. I haven't personally felt the need to add logging.

If there's a benefit, I'm glad to keep it in. If there's not much of a benefit, I'd prefer to drop the logging.

Reply:
Oh sorry it was just part of the process of me figuring out pytest. No logging needed, indeed. I've removed it in [dc3c53d](https://github.com/spdx/ntia-conformance-checker/pull/169/commits/dc3c53df654770de2a6b4015260b7bc2e2ea43ec)